core/src/main/java/org/chatutilities/core/config/MainConfig.java

```diff
@@ -19,6 +20,9 @@ public class MainConfig extends AddonConfig {
   @SwitchSetting
   private final ConfigProperty<Boolean> copy = new ConfigProperty<>(false);
 
+  @TextFieldSetting
+  private final ConfigProperty<String> copyButtonText = new ConfigProperty<>("Copy");
```

Comment:
Could you change this to `CU.get().configuration().getCopyText().get()` such that the default value aligns with the language the user has selected. 
I know there currently is only English, but just future proofing.

Reply:
Hey,
As I am relatively new to laby mod addons. I am not really sure how to do this or the best way this should be done.

Reply:
Heya! No worries, appears to not be possible anyway. The config is initialized before the localization is 🥲